src/wallet/wallet.h

```diff
@@ -310,6 +310,7 @@ class CWalletTx
     enum AmountType { DEBIT, CREDIT, IMMATURE_CREDIT, AVAILABLE_CREDIT, AMOUNTTYPE_ENUM_ELEMENTS };
     CAmount GetCachableAmount(AmountType type, const isminefilter& filter, bool recalculate = false) const;
     mutable CachableAmount m_amounts[AMOUNTTYPE_ENUM_ELEMENTS];
+    mutable bool m_is_cache_empty{true};
```

Comment:
a comment for semantics and uses would be nice

Reply:
@instagibbs done (I think?)